src/App/Model/CryptAES.php

```diff
@@ -37,7 +37,7 @@ public function decrypt($crypt)
     {
         if (!$this->cipher && !$this->setCipher()) return $crypt;
 
-        return $this->cipher->decrypt($crypt);
+        return $this->cipher->decrypt(base64_decode($crypt));
```

Comment:
You db connection is probably not UTF8, have a look here. http://stackoverflow.com/a/11013986/1655741